### ISSUE TITLE
Admins and instructors should be able to mark 'possible plagiarism' revisions as verified or false positives

### ISSUE THREAD

**Issue by ragesoss** [WikiEducationFoundation/WikiEduDashboard#4337]
### Current Behavior:
Revisions that may contain plagiarism are marked with an `ithenticate_id`, the dashboard sends emails to the instructor and Wiki Education staff for the associated course, and users can see lists of revisions that were flagged as possible plagiarism. However, once someone has checked out a revision to see whether it's actually plagiarism or is just a false positive, there's no way to record that result.

### Desired Behavior:
Instructors and admins should be able to mark a revision that was suspected of plagiarism as a verified actual plagiarism, or as a false positive. Ideally, they should be able to add a note about what they determined as well.

### Related Files & Code:

The main place to start is `plagiabot_importer.rb`, which is responsible for identifying possible plagiarism and sending emails for each such revision.

### Additional context:

This data should probably not be stored in the `revisions` table; instead of relying on the (huge) revisions table to keep track of possible plagiarism, a better pattern would be the one we moved to for similar features since the plagiarism checking system was initially implemented: each new case of possible plagiarism should generate an `Alert`, and the `Alert` record can keep track of the status for that revision (unchecked, verified, or false positive).

**Comment by pmartinolli** [comment]
I strongly support this feature because the "possible plagiarism" has a serious chilling effect on the contributors and the solution proposed could mitigate it.

**Comment by ragesoss** [comment]
@pmartinolli I understand that you're coming from the perspective of a having a student whose false positive revision showed up on the public 'Possible Plagiarism' list for a course on Programs & Events Dashboard. Would an alternative label (currently "Possible Plagiarism", or an adjustment to the explanation text ("Wikipedia's plagiarism detection system uses Turnitin/iThenticate to compare new edits with published text. These reports are not 100% accurate. For example, a direct quote with proper attribution may trigger a false positive plagiarism report. Please review the student's contributions and the details of the report before taking any action. (If you find that the change — or the entire article — is not accessible, the content may have been deleted for copyright reasons.)") help mitigate the problem?

The intention is to surface cases where the automated system found matching text that might need to be checked, without presenting it as any kind of accusation. We'll eventually add support for doing more with these than just listing them, but in the short term, changing the language would be easy.

**Comment by pmartinolli** [comment]
Yes, such an adjustement of the explanation text is a first step to mitigate the issue. Another idea : making the warning only visible for the administrators of the event ?

**Comment by ragesoss** [comment]
Do you have an idea for what to adjust in the text? I wrote it with the intention of avoiding this kind of problem, but clearly that didn't work.

The list of revisions that may have copied text is useful to have public, so that other editors who aren't running the event can easily check them.
